src/olympia/migrations/1001-drop-users-picture-type-not-null.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `users`
+MODIFY `picture_type` varchar(75) NULL DEFAULT NULL;
```

Comment:
This looks good to me - did you test locally with `make update_db`?

No idea about the auto generated SQL - you can ignore it.

Reply:
I ran this using `schematic src/olympia/migrations/` after I was using `make shell` to enable a tty on the container.

Reply:
Also, I don't see if there is `make udpate_db` command. There is `update_docker` though

Reply:
`make update_db` is an inner make command, inside the docker container - you can run it with `docker-compose exec web make update_db` - though it just does `schematic src/olympia/migrations/` anyway so you did the same thing.

Reply:
Oh, awesome! I'll keep a note of that for the subsequent PRs. Thanks :)